src/bucket/Bucket.h

```diff
@@ -181,37 +181,29 @@ class Bucket : public std::enable_shared_from_this<Bucket>,
     static uint32_t getBucketVersion(std::shared_ptr<Bucket> const& bucket);
     static uint32_t
     getBucketVersion(std::shared_ptr<Bucket const> const& bucket);
-    BucketEntryCounters getBucketEntryCounters() const;
+    BucketEntryCounters const& getBucketEntryCounters() const;
     friend class BucketSnapshot;
 };
 
-enum class LedgerEntryTypeAndDurability : uint32_t
-{
-    ACCOUNT = 0,
-    TRUSTLINE = 1,
-    OFFER = 2,
-    DATA = 3,
-    CLAIMABLE_BALANCE = 4,
-    LIQUIDITY_POOL = 5,
-    TEMPORARY_CONTRACT_DATA = 6,
-    PERSISTENT_CONTRACT_DATA = 7,
-    CONTRACT_CODE = 8,
-    CONFIG_SETTING = 9,
-    TTL = 10
-};
-
+enum class LedgerEntryTypeAndDurability : uint32_t;
 struct BucketEntryCounters
 {
-    std::map<LedgerEntryTypeAndDurability, size_t> mEntryTypeCounts;
-    std::map<LedgerEntryTypeAndDurability, size_t> mEntryTypeSizes;
+    std::map<LedgerEntryTypeAndDurability, size_t> entryTypeCounts;
+    std::map<LedgerEntryTypeAndDurability, size_t> entryTypeSizes;
 
     BucketEntryCounters& operator+=(BucketEntryCounters const& other);
 
     template <class Archive>
     void
-    serialize(Archive& ar) const
+    save(Archive& ar) const
```

Comment:
Why did this change from the serialize function?

Reply:
I had some cereal errors after adding counters to the `load` function of `BucketIndexImpl`, which I think I eventually solved by adding the counter map comparison to `BucketIndexImpl<T>::operator==`. I split `serialize` into `save` and `load` as part of that process but it wasn't necessary. I've reverted that change now.